src/components/shared/flyout/flyoutSection.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
This file should not be in the `shared/flyout` folder. The only files here are those that pertain to the flyout itself. Not items that can be passed into the children for the flyout component.